roles/dtc/common/templates/ndfc_create_fabric_1222.j2

```diff
@@ -0,0 +1,188 @@
+{% if (isis.authentication_enable | default(defaults.vxlan.underlay.isis.authentication_enable) | title) == 'True' %}
+  ISIS_AUTH_KEYCHAIN_NAME: {{ isis.authentication_keychain_name | default(omit) }}
+  ISIS_AUTH_KEYCHAIN_KEY_ID: {{ isis.authentication_key_id | default(defaults.vxlan.underlay.isis.authentication_key_id) }}
+{% endif %}
+  ISIS_OVERLOAD_ENABLE: {{ isis.overload_bit | default(defaults.vxlan.underlay.isis.overload_bit) }}
+{% if (isis.overload_bit | default(defaults.vxlan.underlay.isis.overload_bit) | title) == 'True' %}
+  ISIS_OVERLOAD_ELAPSE_TIME: {{ isis.overload_bit_elapsed_time | default(defaults.vxlan.underlay.isis.overload_bit_elapsed_time) }}
+{% endif %}
+  ISIS_P2P_ENABLE: {{ isis.network_point_to_point | default(defaults.vxlan.underlay.isis.network_point_to_point) }}
+{% if (bfd.enable | default(defaults.vxlan.underlay.bfd.enable) | title) == 'True' %}
+  BFD_ISIS_ENABLE: {{ bfd.isis | default(defaults.vxlan.underlay.bfd.isis) }}
+{% endif %}
+{# ------------------------------------------------------ #}
+{# Underlay OSPF Parameters                               #}
+{# ------------------------------------------------------ #}
+{% elif (underlay.general.routing_protocol | default(defaults.vxlan.underlay.general.routing_protocol)) == 'ospf' %}
+  OSPF_AREA_ID: {{ ospf.area_id | default(defaults.vxlan.underlay.ospf.area_id) }}
+  OSPF_AUTH_ENABLE: {{ (ospf.authentication_enable | default(defaults.vxlan.underlay.ospf.authentication_enable) | title) }}
+{% if (underlay.ospf.authentication_enable | default(defaults.vxlan.underlay.ospf.authentication_enable) | title) == 'True' %}
+  OSPF_AUTH_KEY_ID: {{ ospf.authentication_key_id | default(defaults.vxlan.underlay.ospf.authentication_key_id) }}
+  OSPF_AUTH_KEY: {{ ospf.authentication_key | default(omit) }}
+{% endif %}
+{% if (bfd.enable | default(defaults.vxlan.underlay.bfd.enable) | title) == 'True' %}
+  BFD_OSPF_ENABLE: {{ bfd.ospf | default(defaults.vxlan.underlay.bfd.ospf) }}
+{% endif %}
+{% endif %}
+{# ------------------------------------------------------ #}
+{# Underlay IPv4 Parameters                               #}
+{# ------------------------------------------------------ #}
+{% if (underlay.general.manual_underlay_allocation | default(defaults.vxlan.underlay.general.manual_underlay_allocation) | title) == 'False' %}
+  LOOPBACK0_IP_RANGE: {{ ipv4.underlay_routing_loopback_ip_range | default(defaults.vxlan.underlay.ipv4.underlay_routing_loopback_ip_range) }}
+  LOOPBACK1_IP_RANGE: {{ ipv4.underlay_vtep_loopback_ip_range | default(defaults.vxlan.underlay.ipv4.underlay_vtep_loopback_ip_range) }}
+  SUBNET_RANGE: {{ ipv4.underlay_subnet_ip_range | default(defaults.vxlan.underlay.ipv4.underlay_subnet_ip_range) }}
+  ANYCAST_RP_IP_RANGE: {{ ipv4.underlay_rp_loopback_ip_range | default(defaults.vxlan.underlay.ipv4.underlay_rp_loopback_ip_range) }}
```

Comment:
The following two lines represent the only difference between this file and the original `ndfc_create_fabric.j2` file.  We need to discuss alternative ways of versioning changes like this so we don't have to replicate the entire file + the changes.